core/src/main/scala/org/apache/spark/util/random/SamplingUtils.scala

```diff
@@ -108,14 +111,22 @@ private[spark] object SamplingUtils {
 private[spark] object PoissonBounds {
 
   /**
-   * Returns a lambda such that Pr[X > s] is very small, where X ~ Pois(lambda).
+   * Returns a lambda such that
+   * {{{
+   * Pr[X > s]
+   * }}}
+   * is very small, where X ~ Pois(lambda).
    */
   def getLowerBound(s: Double): Double = {
     math.max(s - numStd(s) * math.sqrt(s), 1e-15)
   }
 
   /**
-   * Returns a lambda such that Pr[X < s] is very small, where X ~ Pois(lambda).
+   * Returns a lambda such that
+   * {{{
+   * Pr[X < s]
+   * }}}
+   * is very small, where X ~ Pois(lambda).
```

Comment:
- Java
  <img width="301" alt="2016-11-27 3 24 50" src="https://cloud.githubusercontent.com/assets/6477701/20642397/305ade4e-b451-11e6-9b8a-eb7f38381588.png">


Scaladoc not found